sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -15,6 +15,8 @@ title: SearchServiceClient
 input-file:
 - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c99fbb96d7993daec8135a40681d9d807e3f5751/specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json
 - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c99fbb96d7993daec8135a40681d9d807e3f5751/specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json
+modelerfour:
+  seal-single-value-enum-by-default: true
```

Comment:
Why do we need to add a flag to HLCs for an LLC change?  It feels like these two things should be independent of each other.

Reply:
This flag affects method signatures, what has been a constant becomes a parameter. We probably can handle it differently, but we need to figure out how first.